cmd/uplink/cmd/common_test.go

```diff
@@ -89,11 +89,12 @@ func TestUseOrLoadEncryptionKeyIntoEncryptionAccess(t *testing.T) {
 }
 
 func TestSaveLoadEncryptionKey(t *testing.T) {
-	var inputKey []byte
+	var inputKey string
 	{
-		inputKey = make([]byte, rand.Intn(storj.KeySize)*3+1)
-		_, err := rand.Read(inputKey)
+		randKey := make([]byte, rand.Intn(storj.KeySize)*3+1)
```

Comment:
this does not guarantee that len(randKey) > storj.KeySize. This will make the `inputKey[:storj.KeySize]` portion of the test panic in rare situations. See this related jenkins failure and corresponding fix:
https://build.ops.storj.io/blue/organizations/jenkins/storj/detail/master/145/tests/
https://github.com/storj/storj/commit/7a9742f47b19dc88cbd6e63bb0b332bea458cb1c

Reply:
I was aware of that bug in the test which happened because of later changes that I had to do because they were requested in the review and I forgot to update the tests, an due the randomness the tests didn't always panic.

Meanwhile you're right regarding the fact of the key size, the current assertions don't always use the same length to compare the got and expected value, and if I'm not mistaken the conditional should avoid the previous panics that we had.

see https://github.com/storj/storj/pull/2090/files#diff-ca06aa73e8e63be362ca22ba3aec2222R68